use-timescale/tigerlake.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Integrate data lakes with Tiger Cloud
+excerpt: Unifies the Tiger Cloud operational architecture with data lake architectures. This enables real-time application building alongside efficient data pipeline management within a single system.
+products: [cloud]
+price_plans: [scale, enterprise]
+keywords: [data lake, lakehouse, s3, iceberg]
+---
+
+import IntegrationPrereqsCloud from "versionContent/_partials/_integration-prereqs-cloud-only.mdx";
+
+# Integrate data lakes with $CLOUD_LONG
+
+$LAKE_LONG enables you to build real-time applications alongside efficient data pipeline management within a single 
+system. $LAKE_LONG unifies the $CLOUD_LONG operational architecture with data lake architectures. 
+
+This experimental release is a native integration enabling synchronization between $HYPERTABLEs and relational tables
+running in $SERVICE_LONGs, and Iceberg tables running in [Amazon S3 Tables][s3-tables] in your AWS account. 
+
+## Prerequisites
+
+<IntegrationPrereqsCloud/>
```

Comment:
I think we need to update our prereqs to say "with the real-time analytics enabled" to match the new UI